animations/lib/src/basics/08_hero_animation.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+
+class HeroAnimationDemo extends StatelessWidget {
+  static const String routeName = '/basics/hero_animation';
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      backgroundColor: Colors.lightBlue,
+      appBar: AppBar(),
+      body: GestureDetector(
+        child: Hero(
+          tag: 'hero-page-child',
+          child: Container(
+            height: 50.0,
+            width: 50.0,
+            margin: EdgeInsets.all(10.0),
+            padding: EdgeInsets.all(10.0),
+            decoration:
+                BoxDecoration(shape: BoxShape.circle, color: Colors.white),
```

Comment:
nit: A comma after `Collors.white` will cause this to be formatted a little more vertically.